src/main/java/cinema/shop/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package cinema.shop.service.impl;
+
+import cinema.shop.dao.ShoppingCartDao;
+import cinema.shop.dao.TicketDao;
+import cinema.shop.lib.Inject;
+import cinema.shop.lib.ServiceImpl;
+import cinema.shop.model.MovieSession;
+import cinema.shop.model.ShoppingCart;
+import cinema.shop.model.Ticket;
+import cinema.shop.model.User;
+import cinema.shop.service.ShoppingCartService;
+
+@ServiceImpl
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private TicketDao ticketDao;
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setUser(user);
+        ticket.setMovieSession(movieSession);
+        ticketDao.add(ticket);
```

Comment:
if there is no shopping cart for the user - we still save ticket to the db, is it ok?